### ISSUE TITLE
Samples count

### ISSUE THREAD

**Issue by snarb** [happypepper/DeepHoldem#1]
Hi! How much samples did you use to train the network? What was the huber loss? Can You share the training data to play with it and compare with my? (I have implimented deepStack in c++ now generating samples)

**Comment by happypepper** [comment]
I've added the training details to the readme.
You can also observe the validation losses yourself by going to `Data/Models/NoLimit/<street>` and `th`, then `torch.load('final_cpu.info')` in the interactive terminal.

**Comment by snarb** [comment]
Thanks.  Didn't you measure the loss after the buckets CFR ->hole hands CFR  conversion, not for the buckets?  Bucketing by itself should add extra loss. It is interesting how much.

**Comment by happypepper** [comment]
Actually, the loss was measured after the hole -> buckets conversion, so I think the real loss will be higher than the reported loss (but I'm not sure). I agree it would be interesting to measure the difference.
